src/sentry_gitlab/plugin.py

```diff
@@ -65,39 +65,54 @@ def get_new_issue_title(self, **kwargs):
 
     def create_issue(self, request, group, form_data, **kwargs):
 
-        url = self.get_option('gitlab_url', group.project)
+        url = self.get_option('gitlab_url', group.project).rstrip('/')
         token = self.get_option('gitlab_token', group.project)
         repo = self.get_option('gitlab_repo', group.project)
         labels = self.get_option('gitlab_labels', group.project)
-        if repo.find('/') == -1:
-            repo_url = str(repo)
-        else:
-            repo_url = str(repo.replace('/', '%2F'))
+        repo_url = str(repo.replace('/', '%2F'))
 
-        gl = Gitlab(url, token)
+        headers = {'Private-Token': token}
+        session = build_session()
 
         try:
-            gl.auth()
-        except GitlabAuthenticationError as e:
-            raise forms.ValidationError(_('Unauthorized: Invalid Private Token: %s') % (e,))
+            response = session.get(
+                url='%s/api/v3/projects/%s' % (url, repo_url),
+                headers=headers,
+                allow_redirects=False,
+            )
+            response.raise_for_status()
+
+            project_id = response.json()['id']
+
```

Comment:
I'll double check this, but I _thought_ you needed to resolve it to an id. If not, that'd be great.